### DISCUSSION TITLE
捷径 - Miao Yu | 于淼

### DISCUSSION THREAD

**Discussion by giscus** [yufree/yufree.cn#468] [General]
# 捷径 - Miao Yu | 于淼

https://yufree.cn/cn/2024/06/29/shortcut/

**Comment by meeeeeeeeo** [comment]
接受概率、机遇这种概念，而不是给所有事情安排个因果，确实是更适合现代社会的思考方式。

不过以上大概率都是博主挥杆/空军中产生的的哲学思考，应持保守态度阅读  🤔

**Reply by yufree** [reply]
嗯，还有一定概率谈钓鱼只是找个吐槽的引子。
